docker-compose/monitor/README.md

```diff
@@ -26,11 +26,16 @@ The following diagram illustrates the relationship between these components:
 
 # Getting Started
```

Comment:
Using or developing?

Reply:
Yeah, good point, I ignored the user who just wants to demo it. I've updated the Makefile and README instructions in: [ba0a3d7](https://github.com/jaegertracing/jaeger/pull/4452/commits/ba0a3d759c91ad1ef6e63da110e509c42eaad9ef).